modules/sr/robot/motor_devices.py

```diff
@@ -2,13 +2,20 @@
 
 
 class MotorBase:
+    """
+        A base class for any type of motor as these are common attributes
```

Comment:
Style: I suggest not indenting the content of a docstring more than the quotes are:
```suggestion
    A base class for any type of motor as these are common attributes
```
(also elsewhere)